From 90ad2d84c92d6b99cfaacc712cfb37154ed27dca Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Edgar=20Hern=C3=A1ndez?= <ehernand@redhat.com>
Date: Fri, 11 Feb 2022 12:20:01 -0600
Subject: Change Kiali's signing key (#37280)

Kiali is now requiring signing keys that are 16, 24 or 32 bytes long.
This adds a padding to the sample signing key to comply with the new
requirement.
---
 manifests/addons/values-kiali.yaml | 2 +-
 samples/addons/kiali.yaml          | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/manifests/addons/values-kiali.yaml b/manifests/addons/values-kiali.yaml
index 418ff3dd26..bf8a7523f1 100644
--- a/manifests/addons/values-kiali.yaml
+++ b/manifests/addons/values-kiali.yaml
@@ -8,4 +8,4 @@ deployment:
   - '**'
   ingress_enabled: false
 login_token:
-  signing_key: CHANGEME
+  signing_key: CHANGEME00000000
diff --git a/samples/addons/kiali.yaml b/samples/addons/kiali.yaml
index a43a4f40a3..6c09b1db5f 100644
--- a/samples/addons/kiali.yaml
+++ b/samples/addons/kiali.yaml
@@ -105,7 +105,7 @@ data:
       clustering:
         enabled: true
     login_token:
-      signing_key: CHANGEME
+      signing_key: CHANGEME00000000
     server:
       metrics_enabled: true
       metrics_port: 9090
@@ -468,7 +468,7 @@ spec:
         app.kubernetes.io/part-of: "kiali"
         sidecar.istio.io/inject: "false"
       annotations:
-        checksum/config: 0b7a16e51830df3ef2ff816eb3d43341a23c822b38fea95050b3707ba2ca119e
+        checksum/config: a714a9c0c6be44fd505e291c57c8b03f6be5982e66c5887b4e9ffbdf4eb0e1b6
         prometheus.io/scrape: "true"
         prometheus.io/port: "9090"
         kiali.io/dashboards: go,kiali
-- 
2.35.3

